### PR TITLE
Feat/unsupported region for service

### PR DESCRIPTION
**Pull request by zagronitay**
Relates OR Closes #0000

Output from acceptance testing:
More information about running the tests [here](../docs/contributing/e2e_tests.md)

```
$ make testName=TestAccXXX e2e-test-with-apply
...
```

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -39,3 +39,25 @@ jobs:
       - name: Fail if docs are changed
         run: |
           test "$(git status -s ./docs/tables | wc -l)" -eq 0
+
+
+  enpoints:
```
Comment (disq) [inline .github/workflows/lint.yml:44]:
```suggestion
  endpoints:
```

File: client/helpers.go [hunk 2]
```diff
@@ -14,99 +13,86 @@ import (
 	"github.com/aws/smithy-go"
 )
 
-//log-group:([a-zA-Z0-9/]+):
+const (
+	PartitionServiceRegionFile = "data/partition_service_region.json"
+	defualtPartition           = "aws"
```
Comment (disq) [inline client/helpers.go:18]:
```suggestion
	defaultPartition           = "aws"
```

File: client/multiplexers.go [hunk 3]
```diff
@@ -11,30 +11,42 @@ func AccountMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
...
-		for _, region := range client.regions {
-			l = append(l, client.withAccountIDAndRegion(accountID, region))
+func ServiceAccountRegionMultiplexer(service string) func(meta schema.ClientMeta) []schema.ClientMeta {
+	return func(meta schema.ClientMeta) []schema.ClientMeta {
+		var l = make([]schema.ClientMeta, 0)
+		client := meta.(*Client)
+		for accountID := range client.ServicesManager.services {
+			for _, region := range client.regions {
+				if !isSupportedServiceForRegion(service, region) {
+					meta.Logger().Debug("region is not supported for service", "service", service, "region", region)
```
Comment (disq) [inline client/multiplexers.go:21]: oof lots of debug lines. maybe trace level?

**Review by roneli — APPROVED** [review]
LGTM, we do require to split the workflow so it will only run on main

**Review by disq — APPROVED** [review]

**Review by roneli — APPROVED** [review]

**Pull request merged**